LottieSample/build.gradle

```diff
@@ -4,7 +4,7 @@ apply plugin: 'kotlin-android-extensions'
 
 android {
   compileSdkVersion 27
-  buildToolsVersion "27.0.2"
+  buildToolsVersion "27.0.3"
```

Comment:
This should really be done as a separate PR

Reply:
Got it, thanks for your suggestion.